pkg/controller/llmisvc/DEV.md

```diff
@@ -336,11 +336,14 @@ cat<<EOF |oc create -f -
 apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
+  annotations:
+      olm.providedAPIs: CertManager.v1alpha1.operator.openshift.io,Certificate.v1.cert-manager.io,CertificateRequest.v1.cert-manager.io,Challenge.v1.acme.cert-manager.io,ClusterIssuer.v1.cert-manager.io,Issuer.v1.cert-manager.io,IstioCSR.v1alpha1.operator.openshift.io,Order.v1.acme.cert-manager.io
```

Comment:
I missed it in the previous PR, but why do we need `cert-manager` if all our secrets are handled by `service.beta.openshift.io/inject-cabundle: "true"`?

Are we expecting customers to have cert-manager used?